go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Shopify/goose
 
-go 1.15
+go 1.17
```

Comment:
The version in the `go.mod` is the minimum version. I'm okay with that, I think go1.17 is a good minimum nowadays, but that means that the testing matrix below 1.17 is not really useful

Reply:
I'm not sure there is much value in forcing every Goose users to upgrade to 1.17.

Staying compatible does not cost much, here.